src/pages/orderCancelled/OrderCancelled.vue

```diff
@@ -1,34 +1,59 @@
 <template>
   <!-- breadcrum -->
-  <div class="py-4 container flex gap-3 items-center">
-    <router-link :to="{ name: 'home' }" class="text-primary text-base">
-      <i class="fas fa-home"></i>
-    </router-link>
-    <span class="text-sm text-gray-400"><i class="fas fa-chevron-right"></i></span>
-    <p class="text-gray-600 font-medium uppercase">Shopping Cart</p>
+  <div
+    class="flex flex-row items-center justify-start w-1/3 md:w-1/2 sm:w-1/2 py-4 gap-3"
+  >
+    <Breadcrumb :breadcrumbs="breadcrumbList" />
   </div>
   <!-- breadcrum end -->
 
   <!-- order complete wrapper -->
-  <div class="max-w-3xl mx-auto px-4 pt-16 pb-24 text-center">
-    <h2 class="text-4xl font-bold text-primary mb-2">YOUR ORDER IS CANCELLED!</h2>
-    <p class="text-gray-600">We're sorry to see that you have canceled your payment.</p>
-    <p class="text-gray-600">
-      If you need any assistance or have questions about your order, please don't hesitate to contact our support team.
-      We're here to help!
-    </p>
-    <div class="mt-10">
-      <RouterLink
-        to="checkout"
-        class="bg-primary border border-primary text-white px-6 py-3 font-medium rounded-md uppercase hover:bg-transparent hover:text-primary transition text-center"
-      >
-        Back to checkout
-      </RouterLink>
+  <div
+    class="container flex items-center sm:items-start justify-around py-3 gap-3 col-span-12"
+  >
+    <ProfileMenu :username="User.userName" :avatar="User.avatar" />
```

Comment:
đặt tên biến ko viết hoa chữ cái đầu
```suggestion
    <ProfileMenu :username="user.userName" :avatar="user.avatar" />
```